lancero_source.go

```diff
@@ -37,7 +37,7 @@ type LanceroSource struct {
 	active            []*LanceroDevice
 	chan2readoutOrder []int
 	lastFbData        []RawType // used in mix calculation
-	MixFraction       []float64
+	Mix               []Mix
```

Comment:
I like the idea of the slice of objects that can recall both the amount of mix AND the necessary previous value.